portal/models/assessment_status.py

```diff
@@ -278,7 +276,7 @@ def __organization(self):
         for org in self.user.organizations:
             org_rp = org.research_protocol(self.as_of_date)
             if org_rp and org_rp.id == rp_id:
-                return OrgTree().find_top_level_org([org])[0]
+                return OrgTree().find_top_level_orgs([org], first=True)
```

Comment:
Is this just masking the `Indeterminate org lookup` issue by picking the first one arbitrarily?

Reply:
no differently than it was before - that issue remains.